guides/introduction/quick_start_guide.md

```diff
@@ -114,3 +114,189 @@ This will lead you to a web page with a search bar that will allow you to search
+- The `<script>` containing JavaScript initializes `Vue.js`. The code creates a new `Vue` instance that is mandatory to link `Vue.js` with the `DOM`.
+
+To use `instant-meilisearch` in `Vue.js` using `npm` or `yarn` please visit [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue).
+
+:::
+
+::: tab React
+
+The following code sample uses [React](https://reactjs.org/) framework.
+
+```html
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
+</head>
+<html>
+  <body>
+      <div id="app" class="wrapper"></div>
+  </body>
+  <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+  <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+  <script src="https://cdn.jsdelivr.net/npm/react-instantsearch-dom@6.7.0/dist/umd/ReactInstantSearchDOM.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+  <script>
+    const { InstantSearch, SearchBox, Hits, Highlight, Configure }  = ReactInstantSearchDOM;
+    const searchClient = instantMeiliSearch(
+      "http://localhost:7700"
+    );
+
+    const App = () => (
+      React.createElement(InstantSearch, {
+        indexName: "movies",
+        searchClient: searchClient
+      }, [
+        React.createElement(SearchBox, { key: 1 }),
+        React.createElement(Hits, { hitComponent: Hit, key: 2 }),
+        React.createElement(Configure, { hitsPerPage: 10 })]
+      )
+    );
+    function Hit(props) {
+        return React.createElement(Highlight, {
+          attribute: "title",
+          hit: props.hit
+        })
+    }
+    const domContainer = document.querySelector('#app');
+    ReactDOM.render(React.createElement(App), domContainer);
+  </script>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The `<body>` of the page is the entry point for React. `instant-meilisearch` adds the search bar and search results here by manipulating the DOM.
+- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch` in [React](https://reactjs.org/).
+- The `<script>` containing JavaScript initalizes React and contains the code that will be rendered in the body. Customization of `instant-meilisearch` happens here as well.
+
+To use `instant-meilisearch` in `React` using `npm` or `yarn` please visit [meilisearch-react](https://github.com/meilisearch/meilisearch-react).
+
+:::
+
+::::
+
+`instant-meilisearch` uses [instantSearch](https://github.com/algolia/instantsearch.js), an open-source library that generates everything you need for a search interface.
```

Comment:
It looks weird to me to put it at the end: visually it does not look like a footer or a note, but like the latest sentence at the end of the getting started, except the content of this sentence is clearly not a way to close the getting started.
For me, there is no problem to put it in the description of instant-meilisearch before the `Let's Try!` title.

Reply:
https://deploy-preview-553--dreamy-wiles-8fd85b.netlify.app/guides/introduction/quick_start_guide.html#integrate-to-your-project

Reply:
But if both of you think this is ok, let's merge it 😇